dictionaries/dict_merger_svndigger_raft.py

```diff
@@ -2,46 +2,48 @@
 """
 2013/05/08 - Bharadwaj Machiraju (@tunnelshade) - Initial merge script creation
 """
-import os, urllib, codecs
+import os
+import urllib
+import codecs
 
-#Order of the files in the list is important
+# Order of the files in the list is important
 raft_lowercase = [
-                    'raft-small-directories-lowercase.txt',
-                    'raft-small-files-lowercase.txt',
-                    'raft-medium-directories-lowercase.txt',
-                    'raft-medium-files-lowercase.txt',
-                    'raft-large-directories-lowercase.txt',
-                    'raft-large-files-lowercase.txt'
-                ]
+    'raft-small-directories-lowercase.txt',
+    'raft-small-files-lowercase.txt',
+    'raft-medium-directories-lowercase.txt',
+    'raft-medium-files-lowercase.txt',
+    'raft-large-directories-lowercase.txt',
+    'raft-large-files-lowercase.txt'
+]
 
 raft_mixedcase = [
-                    'raft-small-directories.txt',
-                    'raft-small-files.txt',
-                    'raft-medium-directories.txt',
-                    'raft-medium-files.txt',
-                    'raft-large-directories.txt',
-                    'raft-large-files.txt'
-                ]
+    'raft-small-directories.txt',
+    'raft-small-files.txt',
+    'raft-medium-directories.txt',
+    'raft-medium-files.txt',
+    'raft-large-directories.txt',
+    'raft-large-files.txt'
+]
 
-case_dict = {'lowercase':raft_lowercase,'mixedcase':raft_mixedcase}
```

Comment:
Separate the file paths as for example:

`raft_path = os.path.join(abs_path, 'restricted', 'raft')`